b2sdk/file_version.py

```diff
@@ -206,7 +206,7 @@ def __init__(
         content_disposition: Optional[str],
         content_length: int,
         content_language: Optional[str],
-        expires: Optional[datetime],
+        expires: Optional[str],
```

Comment:
why?

Reply:
Parsing to datetime didn't work as it was implemented, and in general I've found it hard to parse these timestamps using builtin python libraries